odometry_correction_node:
  ros__parameters:
    world_frame: "world"
    map_frame: "map"
    odom_frame: "odom"
    base_frame: "robot/base_footprint"
    detection_base_frame: "arena_perception/robot/base_footprint"
    publish_rate: 60.0
    lookup_timeout: 0.1
    log_level: "INFO"